### PR TITLE
Updated regex, keys and user interface 

### PR DESCRIPTION
**Pull request by kezuk23**
It should be a fairly simple one to review. The comments reflect the changes I've made today so let me know if you have any questions.

### CODE REVIEW

File: src/main/java/server/Address.java [hunk 1]
```diff
@@ -2,18 +2,16 @@
 
 public class Address {
 	private int addressID;
-	private String addressNumber; // String variable since address number could contain letters (ie. flat 2B)
```
Comment (m-c-hill) [inline src/main/java/server/Address.java:5]: Good shout removing addressNumber!

**Review by m-c-hill — APPROVED** [review]

File: src/main/java/server/Address.java (hunk 1 above)
Comment (kezuk23) [reply src/main/java/server/Address.java:5]: Thanks! Glad it makes sense!

File: src/main/java/server/InputProcessor.java [hunk 2]
```diff
@@ -11,18 +11,17 @@
...
-    // Add the rest of the keys with their relevant regexes to this Map
-    // The regexes could use some refinement for more precision
     private static final Map<String, String> InfoRegexMap = Map.of(
-        "email", "^(.+)@(.+)$",
-        "date", "^\\d{2}/\\d{2}/\\d{4}$",
-        "letters", "^[A-Za-z]+$",
-        "numbers", "^[0-9]+$",
-        "phonenumber", "^0[\\d]{10}$", //must start with a 0 and be followed by 10 numbers (basic version) OR 1?[\\s-]?\\(?(\\d{3})\\)?[\\s-]?\\d{3}[\\s-]?\\d{4} for international formats (advanced)
-        "postcode", "^[a-zA-Z0-9]*$"
+        "letters", "^[A-Za-z ]+$",
```
Comment (AllenNinous) [inline src/main/java/server/InputProcessor.java:18]: It's very nice seeing that the regexes now account for spaces as well. Well done!

**Review by AllenNinous — APPROVED** [review]
This is very nicely done @kezuk23. Good job!

**Review by Ecenursahin — APPROVED** [review]
Looks great Simon thank you for making the REGEX updates!

**Pull request merged**